indicators/tokenup-crypto-scam.yml

```diff
@@ -0,0 +1,32 @@
+title: Solana cryptocurrency wallet drainer - tokenup
+
+description: |
+    Detects a Solana cryptocurrency wallet drainer that fakes the number
+    of minted NFTs to initiate Fear of Missing Out (FOMO) against the victim.
+
+author: iamdeadlyz
```

Comment:
FYI ℹ️ At some point I'll probably remove usages of the `author` field and replace it with just a call to the GitHub API to automatically keep track of contributors to each rule and display them on phish.report/IOK 🙂